docs/user/getting-started.md

```diff
@@ -166,7 +166,18 @@ $ FC_CLI_BINARY=flightctl-linux-amd64
 Download the `flightctl` binary to your machine:
 
 ```console
-$ curl -LO https://github.com/flightctl/flightctl/releases/download/latest/${FC_CLI_BINARY}
+$ curl -LO https://github.com/flightctl/flightctl/releases/latest/download/${FC_CLI_BINARY}
+
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
+100 29.9M  100 29.9M    0     0  5965k      0  0:00:05  0:00:05 --:--:-- 7341k
+```
+
+To download a `pre-release` use its name instead of `latest`. For example to download `v0.4.0-rc3`:
+
+```console
+$ curl -LO https://github.com/flightctl/flightctl/releases/v0.4.0-rc3/download/${FC_CLI_BINARY}
```

Comment:
This isn't a valid link. You need to keep the order as in the current README:

`curl -LO https://github.com/flightctl/flightctl/releases/download/v0.4.0-rc3/${FC_CLI_BINARY}`

Reply:
`13:37 $ curl -LO https://github.com/flightctl/flightctl/releases/download/v0.4.0-rc3/${FC_CLI_BINARY}
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 24.0M  100 24.0M    0     0  13.3M      0  0:00:01  0:00:01 --:--:-- 29.8M
13:40 $ chmod a+x flightctl-linux-amd64
13:40 $ ./flightctl-linux-amd64 version
flightctl version: v0.4.0-rc3
`
It is valid...

Reply:
Right. But you've added a different link and that gives

```
$ curl -LO https://github.com/flightctl/flightctl/releases/v0.4.0-rc3/download/${FC_CLI_BINARY}
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     9  100     9    0     0     58      0 --:--:-- --:--:-- --:--:--    58

$ cat flightctl-linux-amd64 
Not Found
```